aim/sdk/objects/io/wavfile.py

```diff
@@ -0,0 +1,715 @@
+"""
+NOTICE: All credits to author of SciPy package
```

Comment:
@gorarakelyan here we use the code from scipy package. It's distributed under GPL license.
https://github.com/scipy/scipy/blob/master/LICENSE.txt
Could you please confirm that the copyright notice here is OK? If not, what should be added/changed.

Reply:
I guess it is ok. 

>The Free Software Foundation considers the Apache License, Version 2.0 to be a free software license, compatible with version 3 of the GPL.

Source: https://www.apache.org/licenses/GPL-compatibility.html